assets/js/modules/analytics/common/account-select.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Analytics Account Select component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
```

Comment:
Regarding all these imports of `useSelect` and `useDispatch` - shouldn't we export these from `googlesitekit-data` instead and then import from there? `googlesitekit-data` is where our registry is, and it gets bundled into a completely separate file, so how would the `@wordpress/data` `useSelect` know which registry to look at?

I think we should avoid imports from `@wordpress/data` overall and instead always get such references from our own `googlesitekit-data` to make sure there is no weird edge-case here and that it works properly when bundled. Only `googlesitekit-data` / `googlesitekit/data` (and maybe some tests) should directly utilize anything from `@wordpress/data`.

cc @tofumatt